.github/workflows/build_download_site_index_files.yml

```diff
@@ -53,11 +54,6 @@ jobs:
     container:
       image: ghcr.io/newrelic/s3indexer
     steps:
-      - name: Harden Runner
```

Comment:
Did the Harden Runner step cause a problem? Or was the main problem, the missing permission?

Reply:
`harden-runner` doesn't support being run inside a docker container in a Github action so I removed it.

Reply:
https://github.com/newrelic/newrelic-dotnet-agent/actions/runs/4671862007/jobs/8273364090#step:3:10